airflow/www/views.py

```diff
@@ -1546,7 +1546,8 @@ def tries(self, session=None):
             for ti in task.get_task_instances(start_date=min_date, end_date=base_date):
                 dttm = wwwutils.epoch(ti.execution_date)
                 x.append(dttm)
-                y.append(ti.try_number)
+                # https://issues.apache.org/jira/browse/AIRFLOW-2143
```

Comment:
Not a fan of adding a jira link here. I would rather put an comment here explaining why we are using `ti.prev_attempted_tries` instead of `ti.try_number`.

Reply:
Done.